staging/src/k8s.io/client-go/tools/cache/shared_informer.go

```diff
@@ -409,7 +409,7 @@ func (v *dummyController) HasSynced() bool {
 }
 
 func (v *dummyController) LastSyncResourceVersion() string {
-	return ""
+	return v.informer.LastSyncResourceVersion()
```

Comment:
@deads2k I tried to put this behind feature fate so that it can return empty if the feature is not enabled. But CI is failing with following error:

```shell
$ ./hack/verify-vendor.sh 
+++ [0220 15:21:23] logfile at /tmp/update-vendor.Re5G/update-vendor.log
+++ [0220 15:21:23] go.mod: update staging references
+++ [0220 15:21:31] go.mod: propagate to staging modules
+++ [0220 15:21:32] go.mod: sorting staging modules
+++ [0220 15:22:19] go.mod: tidying
++ date '+[%m%d %H:%M:%S]'
+ timestamp='[0220 15:22:24]'
+ echo '!!! [0220 15:22:24] 1 disallowed k8s.io/apiextensions-apiserver -> k8s.io/kubernetes dependencies exist via the following imports: # k8s.io/kubernetes/pkg/features
k8s.io/apiextensions-apiserver/examples/client-go/pkg/client/informers/externalversions
k8s.io/client-go/tools/cache
k8s.io/kubernetes/pkg/features'
!!! [0220 15:22:24] 1 disallowed k8s.io/apiextensions-apiserver -> k8s.io/kubernetes dependencies exist via the following imports: # k8s.io/kubernetes/pkg/features
k8s.io/apiextensions-apiserver/examples/client-go/pkg/client/informers/externalversions
k8s.io/client-go/tools/cache
k8s.io/kubernetes/pkg/features
+ shift
An error has occurred. Please see more details in /tmp/update-vendor.Re5G/update-vendor.log
Removing /mount/d/repos/github.com/nilekhc/kubernetes/_tmp/kube-vendor.GWuwHv
```

Is there any other way?

Reply:
> Is there any other way?

I'm expecting a featuregate wired like https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/features/known_features.go#L39.  These are controllable by env var in naive binaries and by flag in kube-like binaries.

Reply:
That's neat. Thanks